markdown/preface.markdown

```diff
@@ -164,9 +164,10 @@ scientific computing.
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
+(Spoiler alert: it is.)
 
 At the end of 2008, the Python core developers released Python 3.0 and, as
-Douglas Adams observed tongue-in-cheek about the creation of the Universe in
+Douglas Adams observed about the creation of the Universe in
```

Comment:
@stefanv we added the whole sentence, but I think you can have `tongue-in-cheek` or `about the creation of the Universe`, but not both. It becomes both unwieldy and "explaining the obvious". I also made our stated position more obvious in the opening paragraph of the section.